core/domain/stats_services.py

```diff
@@ -40,6 +40,31 @@ def get_top_unresolved_answers_for_default_rule(exploration_id, state_name):
     }
 
 
+def get_exp_wise_unresolved_answers_count_for_default_rule(exp_ids):
+    exploration_objects = exp_services.get_multiple_explorations_by_id(exp_ids)
+    explorations_ids_list = [
```

Comment:
This part is slightly complicated, so could you please add a comment explaining the contents of exploration_ids_list (i.e. that it contains the exp id repeated number of states times)?